src/intfuncs.h

```diff
@@ -47,7 +47,14 @@ Int HASHKEY_WHOLE_BAG_NC (Obj obj, UInt4 seed);
 // Does NOT perform bounds checking
 Int HASHKEY_BAG_NC (Obj obj, UInt4 seed, Int skip, int read);
 
-Obj IntStringInternal( Obj string );
+// Parse a string containing a decimal integer into a GAP integer
+// object. Returns 'fail' if the input is not a valid decimal.
+// This function ignores leading spaces, and handles signs.
```

Comment:
The statement "This function ignores leading spaces" doesn't seem to be true, either before or after this PR?

Reply:
You’re right, will fix